recipes/sox/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+./configure --prefix=$PREFIX --exec-prefix=$PREFIX
+make
+make bindir=. installcheck
```

Comment:
That means no `make check`? I am not familiar with that syntax. (Due to the high traffic of messages in conda-forge I'd appreciate if you can be more verbose rather than copying and pasting a line.)